app/src/app.rs

```diff
@@ -119,9 +119,9 @@ impl App {
 		let manager = &mut self.cx.manager;
 		let tasks = &mut self.cx.tasks;
 		match event {
-			Event::Cd(url) => {
+			Event::Cd(url, backstack_push) => {
```

Comment:
`backstack_push` seems to be useless anymore?

Reply:
No, it is still necessary. Because when we `cd` via back and forward, we don't want the path to be pushed to stack.